app/src/components/ConvictionVisuals.js

```diff
@@ -292,15 +251,19 @@ export function ConvictionCountdown({ proposal, shorter }) {
 }
 
 export function ConvictionTrend({ proposal }) {
+  const { appState } = useAragonApi()
+  const {
+    globalParams: { alpha },
+  } = appState
   const theme = useTheme()
   const { stakes, max } = getStakesAndThreshold(proposal)
   const blockNumber = useBlockNumber()
-  const { alpha } = getGlobalParams()
   const trend = getConvictionTrend(stakes, max, blockNumber, alpha, TIME_UNIT)
   const { layoutName } = useLayout()
   const compactMode = layoutName === 'small'
-  const percentage =
-    trend > 0.1 ? Math.round(trend * 100) : Math.round(trend * 1000) / 10
+  const percentage = trend.gt(new BigNumber('0.1'))
+    ? Math.round(trend.toNumber() * 100)
+    : Math.round(trend.toNumber() * 1000) / 10
```

Comment:
Was wondering if it makes sense to return the conviction trend as `number` since we already know it's a number between `[-1, 1]` and besides we don't really use it for much else than normal `number` operations

Reply:
i would really prefer to have everything as BN , i don't see any problem with having this as bn and we are sure that we are handling the same data type